app/javascript/packs/report.js

```diff
@@ -52,11 +52,7 @@ $(document).ready(function() {
     caption: "Columns",
     title: "Choose Columns",
     buttonicon: 'ui-icon-script',
-    onClickButton() { return $('#report_grid').jqGrid('columnChooser', {
-      done(stuff) { return (
-        this.resize()
-      ); }
-    }); }
+    onClickButton() { $('#column_selector').toggleClass('hidden'); }
```

Comment:
We don't need to involve jQuery here:
`document.getElementById('column_selector').classLIst.toggle('hidden')`